bfvmm/src/vmcs/test/test_vmcs_intel_x64.cpp

```diff
@@ -43,6 +80,16 @@ write_msr(uint32_t msr, uint64_t val)
 static bool
 vmread(uint64_t field, uint64_t *val)
 {
+    if (VMCS_VM_INSTRUCTION_ERROR == field)
+    {
+        vmcs_vm_instruction_error_read_count++;
+
+        if (vmcs_vm_instruction_error_read_count == 2)
```

Comment:
This is really over-specified.... would love to see an alternative to this. Specifically, if the implementation changes, this unit test will fail even if the code is still valid.